install_script.sh.template

```diff
@@ -397,6 +418,11 @@ elif [ -n "$host_injection_enabled" ] || [ -n "$docker_injection_enabled" ]; the
   apm_libraries+=("datadog-apm-library-all")
 fi
 
+no_config_change=
+if [ -n "$DD_NO_CONFIG_CHANGE" ]; then
```

Comment:
Can we rename this to `DD_APM_NO_CONFIG_CHANGE` or even `DD_APM_INSTRUMENTATION_NO_CONFIG_CHANGE`? This script manipulates multiple configuration files and I think `DD_NO_CONFIG_CHANGE` would be too broad and could be misunderstood.

Reply:
Renamed to `DD_APM_INSTRUMENTATION_NO_CONFIG_CHANGE`.